presto-main/src/main/java/io/prestosql/sql/planner/LocalExecutionPlanner.java

```diff
@@ -2925,6 +2930,48 @@ private OperatorFactory createHashAggregationOperatorFactory(
         }
     }
 
+    private int getDynamicFilteringMaxPerDriverRowCount(Session session, boolean isReplicatedJoin)
```

Comment:
change name to `getDynamicFilteringMaxRowCountPerDriver` (and similar to others)